README.rst

```diff
@@ -57,7 +57,8 @@ Key Benefits
 Compatibility
 =============
 
-llvmlite works with Python 3.8 and greater.
+llvmlite works with Python 3.8 and greater. We attempt to test with the lastest
+Python version that is available in CI.
```

Comment:
```suggestion
llvmlite works with Python 3.8 and greater. We attempt to test with the latest
Python version, this can be checked by looking at the public CI builds.
```
Perhaps this?

Reply:
Sure.